wx_test_paste_census.cpp

```diff
@@ -66,28 +68,24 @@ wxDataViewCtrl* find_census_list_window()
     wxWindowList::const_iterator z = census_children.begin();
     LMI_ASSERT(z != census_children.end());
 
-    wxDataViewCtrl* const dvc = dynamic_cast<wxDataViewCtrl*>(*z);
-    LMI_ASSERT(dvc);
+    wxGrid* const grid = dynamic_cast<wxGrid*>(*z);
+    LMI_ASSERT(grid);
 
-    return dvc;
+    return grid;
 }
 
-// Retrieve the list model from list window.
+// Retrieve the table from grid window.
 //
-// Precondition: this wxDataViewCtrl must actually use a list model.
-wxDataViewListModel* get_census_list_model(wxDataViewCtrl* dvc)
+// Precondition: this wxGrid must actually use a list model.
```

Comment:
This comment doesn't make sense any more and should probably be just removed. I wonder if it even makes sense to still keep this function, as it's just a simple `grid->GetTable()` now -- probably not.